docs/source/relayer.rst

```diff
@@ -0,0 +1,105 @@
+Cross-chain message relayer
```

Comment:
Well in overview section you only mentioned our l1 resolution relay system but we are also using op chains proof mechanism with every fill, you may also include it here to overview.